sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs

```diff
@@ -28,13 +29,60 @@ static StorageTestBase()
 #endif
         }
 
+        /// <summary>
+        /// Add a static TestEventListener which will redirect SDK logging
+        /// to Console.Out for easy debugging.
```

Comment:
Nit - Update the comment to talk about dumping failures as this no longer dumps all logging.